README.md

```diff
@@ -38,8 +36,10 @@ main directory of this repository.
 | `--email-contains`        | `-e`  | (**OR**) Filter for certain author emails.                                |
 | `--email-equals`          |       | (**OR**) Filter for certain author emails.                                |
 | `--duration`              | `-d`  | The time which may pass between two commits that still counts as working. |
-| `--file-extension`        | `-f`  | (**OR**) Filter loc for certain file extension, e.g.`-f cpp`.             |
+| `--file-extension`        | `-f`  | (**OR**) Filter loc for certain file extension, e.g. `-f cpp`.            |
+| `--git`                   |       | Grab the input data from the local Git history.                           |
```

Comment:
"Grab the input data from the local git repository."

Maybe? Maybe also "Git" instead of "git", it just looks very unusual.. 😅

Reply:
I neither understand the purpose of "maybe" in your comment nor the requested change.  It already is "Git" instead of "git" because the latter one is too unusual since Git has the same status as GCC has:  both are a standard.

I implemented the lower case spelling as option trigger anyway since options are usually given as lower case words or phrases.

Reply:
Yeah, sorry. I was rambling a bit.

The text in quotes was a proposal. And then I debated a bit about the Git capitalization, but you are right.

I proposed the change for the same reason, as I said in another comment:

> I think "Parses the Git history" is quite colloquial and maybe not even correct. I have never heard git commits being called the git history. If you have something, that does call it that, I'd be interested. 😄

So, if you find an example there, it would also resolve this.

Reply:
The official Git introduction by Scott Chacon and Ben Straub calls the set of previous commits the "Git history".  There are chapters about "Rewriting the History" in the book, for instance, so I oriented on it when suggesting the new wording.

Reply:
The book is named "Pro Git", published by Apress.  The latest printed version should be the 2nd edition.

Reply:
Great. Then I'll merge this request. 😄